### DISCUSSION TITLE
Need help with vite plugin regarding sitemap

### DISCUSSION THREAD

**Discussion by CurryAyam** [quasarframework/quasar#14769] [General - Components / Directives / etc]
Hi All,

i need to generate a sitemap for my web app in quasar, and i'm having problem to understand how to use this package inside quasar.
https://github.com/jbaubree/vite-plugin-pages-sitemap

could someone help me or guide me.

**Comment by edund** [comment]
I'm having the same problem, can anyone help us?

**Reply by CurryAyam** [reply]
Hi. i've waited for someone to reply me.it's been days.anyway i found this article on the net.
https://stavrovski.net/posts/how-do-i-generate-my-sitemap-xml-with-dynamic-routes-using-vuejs-quasar-vuerouter-and-nodejs-expressjs

hope this help

**Comment by dradux** [comment]
I also have the same problem, I've not found a solution, workaround, or suitable other direction at this point but there is an open issue on the vite-plugin-sitemap package (https://github.com/jbaubree/vite-plugin-sitemap/issues/88). Perhaps if others add to the discussion it will help.
